test/providers/ipc_level_zero_prov_common.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#ifndef IPC_LEVEL_ZERO_PROV_COMMON_H
+#define IPC_LEVEL_ZERO_PROV_COMMON_H
+
+#include <umf/providers/provider_level_zero.h>
+
+#include "level_zero_helpers.h"
+
+void memcopy(void *dst, const void *src, size_t size, void *context) {
```

Comment:
The header files should NOT contain definitions (implementations) of functions, but they should contain declarations only.

Reply:
Fixed in https://github.com/oneapi-src/unified-memory-framework/pull/574